internal/selfmonitor/flowhealth/rules_test.go

```diff
@@ -31,17 +31,17 @@ func TestMakeRules(t *testing.T) {
 	require.Equal(t, "sum by (receiver) (rate(otelcol_receiver_refused_metric_points{service=\"telemetry-metric-gateway-metrics\"}[5m])) > 0", ruleGroup.Rules[4].Expr)
 
 	require.Equal(t, "TraceGatewayExporterSentData", ruleGroup.Rules[5].Alert)
-	require.Equal(t, "sum by (exporter) (rate(otelcol_exporter_sent_spans{service=\"telemetry-trace-gateway-metrics\"}[5m])) > 0", ruleGroup.Rules[5].Expr)
+	require.Equal(t, "sum by (exporter) (rate(otelcol_exporter_sent_spans{service=\"telemetry-trace-collector-metrics\"}[5m])) > 0", ruleGroup.Rules[5].Expr)
 
 	require.Equal(t, "TraceGatewayExporterDroppedData", ruleGroup.Rules[6].Alert)
-	require.Equal(t, "sum by (exporter) (rate(otelcol_exporter_send_failed_spans{service=\"telemetry-trace-gateway-metrics\"}[5m])) > 0", ruleGroup.Rules[6].Expr)
+	require.Equal(t, "sum by (exporter) (rate(otelcol_exporter_send_failed_spans{service=\"telemetry-trace-collector-metrics\"}[5m])) > 0", ruleGroup.Rules[6].Expr)
 
 	require.Equal(t, "TraceGatewayExporterQueueAlmostFull", ruleGroup.Rules[7].Alert)
-	require.Equal(t, "otelcol_exporter_queue_size{service=\"telemetry-trace-gateway-metrics\"} / otelcol_exporter_queue_capacity{service=\"telemetry-trace-gateway-metrics\"} > 0.8", ruleGroup.Rules[7].Expr)
+	require.Equal(t, "otelcol_exporter_queue_size{service=\"telemetry-trace-collector-metrics\"} / otelcol_exporter_queue_capacity{service=\"telemetry-trace-collector-metrics\"} > 0.8", ruleGroup.Rules[7].Expr)
 
 	require.Equal(t, "TraceGatewayExporterEnqueueFailed", ruleGroup.Rules[8].Alert)
-	require.Equal(t, "sum by (exporter) (rate(otelcol_exporter_enqueue_failed_spans{service=\"telemetry-trace-gateway-metrics\"}[5m])) > 0", ruleGroup.Rules[8].Expr)
+	require.Equal(t, "sum by (exporter) (rate(otelcol_exporter_enqueue_failed_spans{service=\"telemetry-trace-collector-metrics\"}[5m])) > 0", ruleGroup.Rules[8].Expr)
 
 	require.Equal(t, "TraceGatewayReceiverRefusedData", ruleGroup.Rules[9].Alert)
```

Comment:
We keep gateway in alert name ? Would it cause confusion since we have trace-collector and alert name is traceGateway ? Should we rename the alert as well when we rename collector to gateway ?